week-03/main.js

```diff
@@ -3,34 +3,56 @@
 // Given the following array
 var list = [1, 2, 3, 4, 5];
 // Remove one from the beginning of the array and add it to the end
-
-
+list.shift();
+list.push(1);
 // # Objects
 
 // Create an object with key value pairs for
 // `name` - a string,
 // `isAlive` - a boolean,
 // and `hobbies` an array.
 // Assign the object to the variable `user`
+var user = {name: 'margie', isAlive: true, hobbies:[]};
 
 
 
-// Add 'golf' to the `user`'s `hobby` array.
 
+// Add 'golf' to the `user`'s `hobby` array.
+user.hobbies = 'golf';
 
 // # Functions
 
 // Define a function `sayHello` that takes a name string and returns "Hello" plus that name and an exclamation point.
 // Example:
 
 // sayHello('Mer') // "Hello Mer!"
+var sayHello = function(name) {
+    return 'Hello ' + name + '!';
+}sayHello('Mer');
 
 // Define a function `createRange` that takes `min` and `max` as parameters and returns an array with all the numbers
-// between min and max
+// between min MatMand max
 // Example:
 
 // createRange(4, 8) // [4, 5, 6, 7, 8]
 
+// min/max number in an array
+var createRange = [4, 5, 6, 7, 8];
+
+createRange = function() {
+for (var i = 0; i < createRange.length; i++) {
+  if (createRange[i] > max) {
+    max = createRange[i];
+  }
+  if (createRange[i] < min) {
+    min = createRange[i];
+  }
+}return createRange;
```

Comment:
`return` would need to be on its own line. But you're returning the function itself and not a new array.